test/lmo.jl

```diff
@@ -926,3 +926,37 @@ end
     @test norm(res[1] - p) < 1e-6
     @test length(res[6]) < 25
 end
+
+@testset "Ordered Weighted Norm LMO" begin
+    Random.seed!(1564)
+    N = Int(1e3)
+    radius = abs(randn())+1
+    direction = randn(N)
+
+    #norm l1
+    weights = zeros(N)
+    fill!(weights,1)
+    lmo = FrankWolfe.OrderWeightNormLMO(weights,radius)
+    lmo_l1 = FrankWolfe.LpNormLMO{1}(radius)
+    v1 = FrankWolfe.compute_extreme_point(lmo,direction)
+    v2 = FrankWolfe.compute_extreme_point(lmo_l1,direction)
+    @test all(v1 .== v2)
```

Comment:
```suggestion
    @test v1 == v2
```
equality test is defined on arrays